### PR TITLE
[DB] animated feature cards

### PR DESCRIPTION
**Pull request by natemoo-re**
Updates the bento box features section of the `/db` page

### CODE REVIEW

**Review by ElianCodes — APPROVED** [review]
Looking amazing 😍

File: package-lock.json
Comment (itsMapleLeaf) [inline package-lock.json:1]: Should probably delete this 😅

File: src/pages/db/_components/features/AstroEcosystem.astro [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+  function setTransition(event) {
+    const card = event.currentTarget
+    clearTimeout(card.transitionTimeoutId)
+    card.style.transition = `transform ${tiltEffectSettings.speed}ms ${tiltEffectSettings.easing}`
+    card.transitionTimeoutId = setTimeout(() => {
+      card.style.transition = ""
+    }, tiltEffectSettings.speed)
+  }
+</script> -->
```
Comment (itsMapleLeaf) [inline src/pages/db/_components/features/AstroEcosystem.astro]:
```suggestion
```
yeet

Comment (itsMapleLeaf) [reply src/pages/db/_components/features/AstroEcosystem.astro]: oh, whatever was there before is now gone lol

File: package-lock.json
Comment (iann-mathaiya) [reply package-lock.json:1]: 😬 I keep forgetting about it, sorry

Comment (itsMapleLeaf) [reply package-lock.json:1]: This might help! https://www.npmjs.com/package/@antfu/ni

**Comment by itsMapleLeaf** [comment]
@iann-mathaiya @natemoo-re The deploy and checks are failing because of an outdated `pnpm-lock.yaml` file. This file needs to be updated whenever a dependency is added. Run `pnpm install` locally to fix this (or use [ni](https://www.npmjs.com/package/@antfu/ni)) and push your changes

**Review by itsMapleLeaf — APPROVED** [review]
beautiful

**Review by ElianCodes — APPROVED** [review]
Beautiful! Really like the 'easter egg' when you click on export table!

File: src/pages/db/_components/Features.astro [hunk 2]
```diff
@@ -17,35 +16,18 @@ import ZeroLockin from "./features/ZeroLockin.astro"
...
-			</figcaption>
-		</figure>
+		<AstroEcosystem />
 
 		<div class="grid gap-2">
 			<figure
 				class="flex flex-col space-y-4 overflow-hidden rounded-xl border border-[#858B98]/20 bg-[#0C0F19]"
 			>
-				<div class="illustration pointer-events-none isolate select-none pt-4" aria-hidden="true">
+				<div class="pt-4" aria-hidden="true">
```
Comment (natemoo-re) [inline src/pages/db/_components/Features.astro:26]: IMO we should leave the `pointer-events-none` and `select-none` on this card. We're using markup, but it is not interactive content (hence the `aria-hidden`)

```suggestion
				<div class="illustration pointer-events-none isolate select-none pt-4" aria-hidden="true">
```

File: src/pages/db/_components/use-cases/Feedback.astro
Comment (natemoo-re) [inline src/pages/db/_components/use-cases/Feedback.astro:1]: This file doesn't need to be committed

**Review by natemoo-re — COMMENTED** [review]
Amazing work @iann-mathaiya! We discussed a few animation and responsiveness improvements offline, but once those are added, this LGTM!

**Pull request merged**